examples/js/loaders/SVGLoader.js

```diff
@@ -8,6 +8,8 @@ THREE.SVGLoader = function ( manager ) {
 
 	this.manager = ( manager !== undefined ) ? manager : THREE.DefaultLoadingManager;
 
+	this.rootNode = null;
```

Comment:
Instead of adding this here, I would change `parse()` so it returns an object instead of the current `paths` array.

```js
var data = { paths: [], xml: xml.documentElement };
```

Reply:
Okay, updated, does that look right?